SOC/models/manna.py

```diff
@@ -7,17 +7,20 @@
 
 class Manna(common.Simulation):
     """Implements the Manna model."""
-    def __init__(self, L: int, critical_value: int = 1):
+    
+    def __init__(self, L: int, critical_value: int = 1, abelian: bool = True):
         """
         :param L: linear size of lattice, without boundary layers
         :type L: int
         :param critical_value: 1 by default - above this value, nodes start toppling
         :type critical_value: int
+        :param abelian: True by default - abelian, False - nonabelian
+        :type abelian: bool
         """
         super().__init__(L)
         self.values = np.zeros((self.L_with_boundary, self.L_with_boundary), dtype=int)
         self.critical_value = critical_value
-
+        self.abelian = abelian
```

Comment:
wydaje sie lepsza konwencja isabelian

Reply:
Też fajnie brzmi. Jeśli @StanczakDominik nie zawetuje, to zmienimy.:)

Reply:
Izabelowski? ;)

Może być.